docs/src/loop.rst

```diff
@@ -226,6 +226,15 @@ API
        invalid. That function must be called again to determine the
        correct backend file descriptor.
 
+.. c:function:: uv_loop_cb uv_loop_get_watcher_queue_changed_callback(uv_loop_t* loop)
+
+    Returns `loop->on_watcher_queue_updated`.
```

Comment:
Sorry, I just noticed these should have `.. versionadded:: 1.26.0` (you can see other examples in the docs).